routes/deductions/deductions.controller.js

```diff
@@ -215,15 +216,31 @@ module.exports = function(app) {
     '/trillium/longTermCare',
     checkSchema(trilliumlongTermCareSchema),
     checkErrors('deductions/trillium-longTermCare'),
-    doYesNo('trilliumLongTermCareClaim', 'trilliumLongTermCareAmount'),
+    postLongTermCare,
     doRedirect,
   )
+
+  app.get('/trillium/longTermCare/type', renderWithData('deductions/trillium-longTermCare-type'))
+  app.post(
+    '/trillium/longTermCare/type',
+    checkSchema(trilliumLongTermCareTypeSchema),
+    checkErrors('deductions/trillium-longTermCare-type'),
+    doYesNo('trilliumLongTermCareTypeClaim', 'trilliumLongTermCareAmount'),
+    // These only apply if the user clicked "no"
+    // If they clicked "Yes", they will be redirected by `doYesNo()`
+    (req, res, next) => {
+      req.session.deductions.trilliumEnergyAmount = 0
+      next()
+    },
+    doRedirect,
+  )
+
   app.get(
-    '/trillium/longTermCare/amount',
+    '/trillium/longTermCare/type/amount',
     renderWithData('deductions/trillium-longTermCare-amount'),
   )
   app.post(
-    '/trillium/longTermCare/amount',
+    '/trillium/longTermCare/type/amount',
```

Comment:
We're changing the URL here because of 'doYesNo', right?

Reply:
yeah exactly